example_data/sampleFHIR_V2.json

```diff
@@ -0,0 +1,1247 @@
+                {
+                  "system": "http://loinc.org",
+                  "code": "48005-3",
+                  "display": "Amino acid change (pHGVS)"
+                }
+              ]
+            },
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "http://varnomen.hgvs.org/",
+                  "code": "p.(Gly40Val)",
+                  "display": "p.(Gly40Val)"
+                }
+              ]
+            }
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "53034-5",
+                  "display": "zygosity"
+                }
+              ]
+            },
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "LA6706-1",
+                  "display": "heterozygote"
+                }
+              ]
+            }
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "53037-8",
+                  "display": "Clinical significance"
+                }
+              ]
+            },
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "LA26333-7",
+                  "display": "uncertain_significance"
+                }
+              ]
+            }
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "53034-0",
+                  "display": "Inheritance pattern based on family history"
+                }
+              ]
+            },
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "LA24641-5",
+                  "display": "autosomal_recessive_digenic"
+                }
+              ]
+            }
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://www.genenames.org/geneId",
+                  "code": "HGNC:795",
+                  "display": "BRAT1"
+                }
+              ]
+            },
+            "valueString": "BRAT1"
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://hl7.org/evidence",
+                  "code": "evidence",
+                  "display": "Evidence for classification of variant"
+                }
+              ]
+            },
+            "valueString": "These variants occurs in a recessive genes that have been 100% sequenced and no second variant identified."
```

Comment:
this was peviously the ACMG classification evidence

e.g. `absent from the gnomAD population database (PM2_Moderate).affects a gene with a low tolerance for missense variation (PP2_Supporting). predicted to be deleterious by in silico prediction tools (PP3_Supporting). similar variants affecting the same amino acid c.118G>A p.(Gly40Arg), c.118G>C p.(Gly40Arg), c.118G>T p.(Gly40Trp) & c.119G>A p.(Gly40Glu) have been previously reported in the literature (1-2) (PM5_Moderate) reported on ClinVar as likely pathogenic.` I think it should either be the specific evidence or nothing as this doesn't refer specifically to this variant

Reply:
This evidence is associated with a particular gene. These examples vary as well.

Reply:
I guess that's my question. Is this always going to be gene-level information on sequencing, and does it always have to be populated?

Reply:
Sorry for this mixup. Yes, it's not. I was just copying and pasting values to see how it'd come out in RedCap.